docs/developer-guide/serialization.rst

```diff
@@ -80,6 +80,21 @@ If a statement doesn't set the value wrapping explicitly, KSQL uses the system
 default, defined by ``ksql.persistence.wrap.single.values``. You can change the system default.
 For more information, see :ref:`ksql-persistence-wrap-single-values`.
 
+.. important:: KSQL treats ``null` keys and values as a special case. We recommend avoiding
+               unwrapped single-field schemas if the field can have a ``null`` value.
+
+A ``null`` value in a table's topic is treated as a tombstone, which indicates that a row has been
+removed. If a table's source topic has an unwrapped single-field key schema and the value is
+``null``, it's treated as a tombstone, resulting in any previous value for the key being removed
+from the table.
+
+A ``null`` key or value in a stream's topic is ignored when the stream is part of a join.
+A ``null`` key in a table's topic is treated as a tombstone, and a ``null`` value is
```

Comment:
Did you mean the null key is ignored and the null value is tombstone?

Reply:
Good catch. Thanks.